### ISSUE TITLE
[Pipeline View]  Improved pipeline navigation 

### ISSUE THREAD

**Issue by jama-pivotal** [concourse/concourse#1508]
This is a master issue describing changes for the pipeline view in Concourse. The purpose of this issue is to improve navigation between pipelines and back to Dashboard

# Sub-Issues 

- [ ] Pipeline name (ref #1207) 
- [ ] Navigating from pipeline back up to Dashboard 
- [ ] Switching between pipelines 
- [ ] Breadcrumbs (ref #2139)

**Comment by jama-pivotal** [comment]
Waiting on designs for each, will create sub-issues when ready @Lindsayauchin

**Comment by sampeinado** [comment]
@Lindsayauchin and I propose to solve this issue in the following way, corresponding to @jama-pivotal 's four points above. Please find icons attached. 

- [ ] adding the pipeline name with an icon to the navbar
- [ ] allowing the user to click on the propeller logo to navigate to the dashboard
<img width="466" alt="screen shot 2018-04-09 at 12 12 45 pm" src="https://user-images.githubusercontent.com/35495560/38509251-615380d8-3bef-11e8-9b75-11b50d7ab828.png">

- [ ] allowing the user to switch between pipelines via the dashboard
<img width="439" alt="screen shot 2018-04-09 at 12 15 00 pm" src="https://user-images.githubusercontent.com/35495560/38509369-b530b1c6-3bef-11e8-9bce-ea5a16911cb3.png">

- [ ] showing a breadcrumb in the top nav for when the user is on the resource, build or job pages
<img width="365" alt="screen shot 2018-04-09 at 12 14 10 pm" src="https://user-images.githubusercontent.com/35495560/38509309-8a5f2964-3bef-11e8-82f9-088b54a53ddd.png">

Font for breadcrumbs - 18 px, bold, white
Font for forward-slash that divides breadcrumbs - 18 px, bold, white, .33 opacity

You can demo the new interaction flow using [this invision prototype](https://pivotal.invisionapp.com/share/FZGR15XHXE8#/screens/290058350). 

[icons.zip](https://github.com/concourse/concourse/files/1891039/icons.zip)

**Comment by jama-pivotal** [comment]
closing in favor of #2139 and #2282
